synapseclient/api/table_services.py

```diff
@@ -0,0 +1,71 @@
+"""
+The purpose of this module is to provide any functions that are needed to interact with
+columns in the Synapse REST API.
+"""
+
+import json
+from typing import TYPE_CHECKING, List, Optional
+
+if TYPE_CHECKING:
+    from synapseclient import Synapse
+    from synapseclient.models import Column
+
+
+async def get_columns(
+    table_id: str,
+    *,
+    synapse_client: Optional["Synapse"] = None,
+) -> List["Column"]:
```

Comment:
Why is Column in quotes here?

Reply:
When the typing is in quotes it means that it was imported behind this check:

```

if TYPE_CHECKING:
    from synapseclient import Synapse
    from synapseclient.models import Column
```

You have to use quotes because it's only for static type checking. It's Pythons mechanism to allow for typing to work while preventing circular imports from occurring.